libs/@blockprotocol/type-system/rust/src/schema/data_type/constraint/boolean.rs

```diff
@@ -16,7 +19,16 @@ pub enum BooleanTypeTag {
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 pub struct BooleanSchema;
```

Comment:
is `BooleanSchema` supposed to have fields added at a later stage?

Reply:
At least not in the immediate future, no. But it would be possible to add a `const` field to represent something like `{ myFlag: false }` (as a type).